### ISSUE TITLE
add simple style tweak how tos

### ISSUE THREAD

**Issue by jtauber** [pinax/pinax#101]
- [ ] where to go to edit LESS and how to rebuild CSS
- [ ] where to go to edit footer
- [ ] how to change the site name in the top left
- [ ] how to change the top left to be a logo
- [ ] how to add left nav bar

**Comment by jtauber** [comment]
some of thus duplicates #77

**Comment by shoshseiden** [comment]
I am interested in possibly contributing to this issue, if it hasn't been done already. Is there a certain file in the docs where I would contribute this information?

**Comment by ossanna16** [comment]
@shoshseiden I would suggest adding a new file but it would probably be best to discuss this with @jtauber :)

**Comment by jtauber** [comment]
yeah, I think a separate file with a section of each of the howtos I suggest in the description above

**Comment by jtauber** [comment]
@shoshseiden if you're not sure how to do any of the steps, just asking the channel; take notes and put together a pull request for the docs. Thanks!

**Comment by shoshseiden** [comment]
Awesome! Thank you, @jtauber @ossanna16 !
